protobuf/drand/protocol.proto

```diff
@@ -106,6 +106,7 @@ message DKGPacket{
 // chain
 message SyncRequest {
     uint64 from_round = 1;
+    bool follow = 3;
```

Comment:
I think you need to update the watcher client and/or the cli to pass this flag for the relevant command